### PR TITLE
[02] Davydik complete homework-2

### PR DESCRIPTION
**Pull request by elena-davydik**

### CODE REVIEW

File: 02/Elena Davydik/src/components/Main/HotelsSearchForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { ChangeEvent } from "react";
+
+type Props = {
+    onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+}
+
+const searchFormTitle: string = 'Available hotels';
```
Comment (funny-code-404) [inline 02/Elena Davydik/src/components/Main/HotelsSearchForm/index.tsx:7]: you can skip type declaration for primitives

File: 02/Elena Davydik/src/components/Main/HotelsSearchForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+export const HotelsSearchForm = ({onChange}: Props) => (
+    <div className="container">
+        <h2 className="title">{searchFormTitle}</h2>
+        <input onChange = {onChange} placeholder = 'Enter destination or hotel name' className="searchForm__input"/>
```
Comment (funny-code-404) [inline 02/Elena Davydik/src/components/Main/HotelsSearchForm/index.tsx:12]: you declared a variable for string on the line 7, but use hardcoded string for placeholder. move it in the variable as well

File: 02/Elena Davydik/src/components/Main/HotelsSearchForm/index.tsx (hunk 1 above)
Comment (elena-davydik) [reply 02/Elena Davydik/src/components/Main/HotelsSearchForm/index.tsx]: Skip type declaration for variable _searchFormTitle_

File: 02/Elena Davydik/src/components/Main/HotelsSearchForm/index.tsx (hunk 2 above)
Comment (elena-davydik) [reply 02/Elena Davydik/src/components/Main/HotelsSearchForm/index.tsx]: Create the variable _inputPlaceholderText_ for input placeholder

**Pull request merged**